app/pdfs/response_report.rb

```diff
@@ -0,0 +1,74 @@
+class ResponseReport < Prawn::Document
+  def initialize(response)
+    super(page_size: 'A4')
+    @response = response
+
+    header_text 'Inleiding'
+    introduction
+    move_down 20
+
+    creation_date
+    move_down 20
+
+    header_text 'Resultaten'
+    domain_results_table
+    move_down 20
+
+    header_text 'Antwoorden'
+    answers_table
+  end
+
+  private
+
+  def header_text(str)
+    text str, size: 24, style: :bold
+  end
+
+  def creation_date
+    text "Ingevuld op #{I18n.l @response.created_at, format: :long}"
+  end
+
+  def domain_results
+    @response.domain_ids.each do |domain_id|
+      text "Domein: #{domain_id}"
+    end
+  end
+
+  def domain_results_table
+    table_data = @response.domain_ids.map do |domain_id|
+      [domain_id, @response.estimate_interpretation]
+    end
+
+    table table_data, column_widths: [300, 150]
+  end
+
+  def answers_table
+    table_data = @response.questions.map do |question|
+      [question.text, selected_answer_text(question)]
```

Comment:
Bij `question.text` graag ook `question.intro`. Zo is het nu in PR voor de webweergave:
![image](https://cloud.githubusercontent.com/assets/8157262/22285420/e52fcd5c-e2eb-11e6-8ccf-b4f55acbfb5f.png)